Makefile

```diff
@@ -27,6 +29,12 @@ CRD_OPTIONS ?= "crd:trivialVersions=true"
 
 GATEKEEPER_MANIFEST_DIR ?= config/gatekeeper
 
+ifeq (openshift, $(KUBE_DISTRIBUTION))
+RBAC_DIR=config/rbac/overlays/openshift
+else
+RBAC_DIR=config/rbac/base
```

Comment:
Is it possible to keep the original directory as the base i.e. do not move the yaml manifests into a new `base` subdirectory? I think it should be possible, that way we can just have the overlay directory that is unique as I want to make sure things like `make manifests` continue to work properly without modifications.

Reply:
Hi, it's actually not possible to do this - kustomize throws an error if the overlays directory is inside the base directory: 
`Error: accumulating resources: accumulateFile "accumulating resources from '../..': '/Users/wkutler/go/src/github.com/willkutler/gatekeeper-operator/config/rbac' must resolve to a file", loader.New "cycle detected: candidate root '/Users/wkutler/go/src/github.com/willkutler/gatekeeper-operator/config/rbac' contains visited root '/Users/wkutler/go/src/github.com/willkutler/gatekeeper-operator/config/rbac/overlays/openshift'"`

Reply:
I actually prefer your new directory layout as it's more clear what's the base and what's the overlay. But we need to tell `controller-gen` to use the new directory layout. This is possible by adding `output:rbac:dir=config/rbac/base` to the `controller-gen` command when we `make manifests`. Then we should delete the `config/rbac/role.yaml` that is left.